contrib/babelfishpg_tsql/src/tsqlUnsupportedFeatureHandler.cpp

```diff
@@ -1139,6 +1138,20 @@ antlrcpp::Any TsqlUnsupportedFeatureHandlerImpl::visitDdl_statement(TSqlParser::
 		return visitChildren(ctx);
 	}
 
+	if (ctx->create_schema())
+	{
+		auto create_schema = ctx->create_schema();
+		if (create_schema->grant_statement().size() > 0)
```

Comment:
We should check if `create_schema->grant_statement()` and `create_schema->revoke_statement()` are NOT NULL.

Reply:
With `create_schema->grant_statement()` check, it was throwing an error while compilation. That's the reason, we have used `size()` to check it is NULL.
```
src/tsqlUnsupportedFeatureHandler.cpp:1144:37: error: could not convert ‘TSqlParser::Create_schemaContext::grant_statement()()’ from ‘std::vector<TSqlParser::Grant_statementContext*>’ to ‘boo’
   if (create_schema->grant_statement())
       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~
```

Reply:
Ohh cause it's a vector.